# Motor speed in QPPS
int32 m1_qpps
int32 m2_qpps

# Max seconds before automatically stopping
uint32 max_secs
